androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/home/HomeScreen.kt

```diff
@@ -25,14 +25,14 @@ import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyHeaderUiModel
 import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.SurveyUiModel
 import coil.compose.AsyncImage
 import kotlinx.coroutines.launch
-import org.koin.androidx.compose.getViewModel
 
 private const val TopGradientAlpha: Float = 0.01f
 private const val BottomGradientAlpha: Float = 0.6f
 
 @Composable
 fun HomeScreen(
-    homeViewModel: HomeViewModel = getViewModel(),
+    homeViewModel: HomeViewModel,
```

Comment:
I think this will cause an issue when you try to add instrumented test for `HomeScreen` since `viewModel` is instantiated at navigation. Please put it back to the screen level. 🙏

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/118/commits/690207160547acd356edae11b175d058bc1a990f